### PR TITLE
Update run mode in schema

### PR DESCRIPTION
**Pull request by MetRonnie**
Needs
- https://github.com/cylc/cylc-flow/pull/6039

### CODE REVIEW

File: cylc/uiserver/schema.py [hunk 1]
```diff
@@ -170,7 +148,7 @@ class Arguments:
             ''')
         )
         mode = RunMode(
-            default_value=RunMode.Live.name  # type: ignore[attr-defined]
+            default_value=RunMode.Live.name
```
Comment (wxtim) [inline cylc/uiserver/schema.py:151]: Is inheriting `graphene.Enum` a purely convenience thing?

Comment (oliver-sanders) [reply cylc/uiserver/schema.py:151]: It's importing `cylc.flow.network.schema.WorkflowRunMode` ([which is a `graphene.Enum`](https://github.com/cylc/cylc-flow/blob/5b2b1808f1c8f7056efabd2a5a5fe1a93de39690/cylc/flow/network/schema.py#L627-L631)) as `RunMode`

**Review by oliver-sanders — APPROVED** [review]

**Review by wxtim — APPROVED** [review]

**Comment by wxtim** [comment]
Tests look sad

**Comment by MetRonnie** [comment]
#650

**Pull request merged**